code/modules/paperwork/fax.dm

```diff
@@ -0,0 +1,133 @@
+/obj/machinery/fax
```

Comment:
this should have a machine board so they can be built

Reply:
I wrote in the description of the PR why I did not add the board. It's not in **/obj/machinery/photocopier** either.

Reply:
> I also did not envision recreating a fax using the boards, because I am afraid people will start abusing it. 

how would they abuse it? I think it would be really funny to be able to fax the HOP threats from maintenance. I agree with john, it should be printable.

Reply:
> how would they abuse it? I think it would be really funny to be able to fax the HOP threats from maintenance. I agree with john, it should be printable.

Okay, I'll do it.